doc/source/tutorial/stats/rv_infrastructure.md

```diff
@@ -0,0 +1,920 @@
+
+```{code-cell} ipython3
+from scipy import stats
+mu, sigma = 0, 1
+X = stats.Normal(mu=mu, sigma=sigma)
+X
+```
+
+Once instantiated, shape parameters can be read (but not written) as attributes.
+
+```{code-cell} ipython3
+X.mu, X.sigma
+```
+
+Note that the documentation of [`scipy.stats.Normal`](https://scipy.github.io/devdocs/reference/generated/scipy.stats.Normal.html#scipy.stats.Normal) contains links to more detailed documentation of each of its methods. (Compare, for instance, against the documentation of [`scipy.stats.norm`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.norm.html).)
+
+Methods are then called by passing only the argument(s), if there are any, not distribution parameters.
+
+```{code-cell} ipython3
+X.mean()
+```
+
+```{code-cell} ipython3
+X.pdf(x)
+```
+
+For simple calls like this (e.g. the argument is a valid float), calls to methods of the new random variables will typically be faster than comparable calls to the old distribution methods.
+
+```{code-cell} ipython3
+%timeit X.pdf(x)
+```
+
+```{code-cell} ipython3
+%timeit frozen.pdf(x)
+```
+
+```{code-cell} ipython3
+%timeit dist.pdf(x, loc=mu, scale=sigma)
+```
+
+Note that the calls to `X.pdf` and `frozen.pdf` are identical, and the call to `dist.pdf` is very similar - the only difference is that the call to `dist.pdf` includes the shape parameters, whereas in the new infrastructure, shape parameters are only provided when the random variable is instantiated.
+
+Besides `pdf`, several other methods of the new infrastructure are essentially the same as the old methods.
+
++++
+
+- `logpdf` (logarithm of probability density function)
+- `cdf` (cumulative distribution function)
+- `logcdf` (logarithm of cumulative distribution function)
+- `entropy` (differential entropy)
+- `median`
+- `support`
+
++++
+
+Others methods have new names, but are otherwise drop-in replacements.
+- `sf` (survival function) $\rightarrow$ `ccdf` (complementary cumulative distribution function)
+- `logsf` $\rightarrow$ `logccdf`
+- `ppf` (percent point function) $\rightarrow$ `icdf` (inverse cumulative distribution function)
+- `isf` (inverse survive function) $\rightarrow$ `iccdf` (inverse complementary cumulative distribution function)
+- `std` $\rightarrow$ `standard_deviation`
+- `var` $\rightarrow$ `variance`
+
++++
+
+The new infrastructure has several *new* methods in the same vein as those above.
+
+- `ilogcdf` (inverse of the logarithm of the cumulative distribution function)
+- `ilogccdf` (inverse of the logarithm of the complementary cumulative distribution function)
+- `logentropy` (logarithm of the entropy)
+- `mode` (mode of the distribution)
+- `skewness`
+- `kurtosis` (*non-excess* kurtosis; see "Standardized Moments" below)
+
++++
+
+And it has a new `plot` method for convenience
+
+```{code-cell} ipython3
+import matplotlib.pyplot as plt
+X.plot()
+plt.show()
+```
```

Comment:
The default plot title is a bit awkward `Normal(mu=np.float64(0.0), sigma=np.float64(1.0))`. We could add a title manually; in an case, this is not a requirement, just a comment.

Reply:
Actually if @tylerjereddy will allow a backport for this, this is a little regression in the rendering. In a few lines, @steppi could add `__str__` overrides very similar to the existing `__repr__` overrides that would fix this.

Reply:
Since it looks doc-only, I'm in favour of doing that.

Reply:
@mdhaber, would rather I push a commit with the `__str__` overrides here, or make a separate PR against `main`?

Reply:
Resolved in gh-22081